### PR TITLE
Feature/cases/case tab

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

File: src/app/shared/dynamic-form/components/form-null/form-null.component.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@use '/src/variables.scss' as *;
+
+.form-null-label {
+  display: block;
+  height: 1rem;
+  color: rgba(0, 0, 0, 0.6);
```
Comment (sandorbanvitagroup) [inline src/app/shared/dynamic-form/components/form-null/form-null.component.scss:6]: Please use variable for black color

File: src/app/shared/dynamic-form/components/form-null/form-null.component.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.form-null-value {
+  display: block;
+  font-weight: 500;
+}
+
+.form-null-hint {
+  display: block;
+  color: rgba(0, 0, 0, 0.6);
```
Comment (sandorbanvitagroup) [inline src/app/shared/dynamic-form/components/form-null/form-null.component.scss:16]: Same here, a variable would be nice.

File: src/app/shared/dynamic-form/components/form-null/form-null.component.scss (hunk 1 above)
Comment (danielrus) [reply src/app/shared/dynamic-form/components/form-null/form-null.component.scss]: done

File: src/app/shared/dynamic-form/components/form-null/form-null.component.scss (hunk 2 above)
Comment (danielrus) [reply src/app/shared/dynamic-form/components/form-null/form-null.component.scss]: done

**Review by sandorbanvitagroup — APPROVED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**